databroker/src/grpc/kuksa_val_v2/conversions.rs

```diff
@@ -10,331 +10,142 @@
-//                 }
-//                 proto::datapoint::Value::FloatArray(array) => {
-//                     broker::DataValue::FloatArray(array.values)
-//                 }
-//                 proto::datapoint::Value::DoubleArray(array) => {
-//                     broker::DataValue::DoubleArray(array.values)
-//                 }
-//             },
-//             None => broker::DataValue::NotAvailable,
-//         }
-//     }
-// }
-
-// impl From<&broker::Field> for proto::Field {
-//     fn from(from: &broker::Field) -> Self {
-//         match from {
-//             broker::Field::Datapoint => proto::Field::Value,
-//             broker::Field::ActuatorTarget => proto::Field::ActuatorTarget,
-//             broker::Field::MetadataUnit => proto::Field::MetadataUnit,
-//         }
-//     }
-// }
-
-// impl TryFrom<&proto::Field> for broker::Field {
-//     type Error = &'static str;
-
-//     fn try_from(from: &proto::Field) -> Result<Self, Self::Error> {
-//         match from {
-//             proto::Field::Value => Ok(broker::Field::Datapoint),
-//             proto::Field::ActuatorTarget => Ok(broker::Field::ActuatorTarget),
-//             _ => Err("Unknown field"),
-//         }
-//     }
-// }
-
-// impl From<proto::Datapoint> for broker::Datapoint {
-//     fn from(from: proto::Datapoint) -> Self {
-//         Self {
-//             ts: SystemTime::now(),
-//             source_ts: match from.timestamp {
-//                 Some(ts) => match std::convert::TryInto::try_into(ts) {
-//                     Ok(ts) => Some(ts),
-//                     Err(_) => None,
-//                 },
-//                 None => None,
-//             },
-//             value: broker::DataValue::from(from.value),
-//         }
-//     }
-// }
-
-// impl From<broker::EntryUpdate> for proto::DataEntry {
-//     fn from(from: broker::EntryUpdate) -> Self {
-//         Self {
-//             path: from.path.unwrap_or_default(),
-//             value: match from.datapoint {
-//                 Some(datapoint) => Option::<proto::Datapoint>::from(datapoint),
-//                 None => None,
-//             },
-//             actuator_target: match from.actuator_target {
-//                 Some(Some(actuator_target)) => Option::<proto::Datapoint>::from(actuator_target),
-//                 Some(None) => None,
-//                 None => None,
-//             },
-//             metadata: {
-//                 let metadata = proto::Metadata {
-//                     unit: from.unit,
-//                     ..Default::default()
-//                 };
-//                 Some(metadata)
-//             },
-//         }
-//     }
-// }
+impl From<&broker::UpdateError> for proto::Error {
+    fn from(update_error: &broker::UpdateError) -> Self {
+        match update_error {
+            broker::UpdateError::NotFound => proto::Error {
+                code: proto::ErrorCode::NotFound.into(),
+                message: "Not Found".to_string(),
```

Comment:
where do I know here what is not found? Like an id or path maybe? For subscribe or batch actuate?